static/js/containers/ChannelPage.js

```diff
@@ -50,17 +62,26 @@ class ChannelPage extends React.Component {
   }
 
   loadData = () => {
-    const { dispatch, channelName } = this.props
+    const { dispatch, channelName, location: { search } } = this.props
     Promise.all([
```

Comment:
I know this is not your code but is `Promise.all` doing anything here? we aren't using its return value

Reply:
I'd prefer not to touch any more code unless it makes a functional difference at this point.

Reply:
If we don't remove dead code now, when are we supposed to do it? We have unit tests for this to make sure we don't break functionality

Reply:
Generally, I'd prefer to do it as part of addressing tech debt, particularly on a big PR like this. I did it anyway.